docs/react/toggling-devices.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 4
+---
+
+# Toggling devices
+
+Fishjam Cloud provides functionality to control media device streams more dynamically. This includes both soft control, such as muting, and hard control, like physically turning the device on or off.
+
+## Soft Mode - muting and unmuting
+
+Soft Mode allows you to enable or disable the media stream without affecting the device's operational status.  
+If the soft mode is disabled, it starts the device, enables the media stream, and starts (or resumes) streaming.  
```

Comment:
I don't know if 'mode is disabled' is correct 🤔. The device could be disabled, and we could invoke/call/trigger this function when the device is disabled.